src/lib.rs

```diff
@@ -237,6 +239,8 @@
 #[macro_use]
 extern crate std;
 
+pub extern crate ed25519;
```

Comment:
Note: original `ed25519` module renamed to `keypair`, the `ed25519` crate re-exported in its place, and the re-exports for all of the other modules (which were previously in the `ed25519` module) moved below.